### ISSUE TITLE
How do I use the attribute "value"?

### ISSUE THREAD

**Issue by Alezter** [victorybiz/laravel-simple-select#6]
I'm trying to use the select on my edit page, so far I manage to use multiple by passing an array of id to value.

controller.php
```
public function getSelected($role)
{
    $permissions = $role->permissions()->pluck('id');
    $array = [];

    foreach ($permissions as $value) {
        $array[] = $value;
    }

    return $array;
}
```

edit.blade.php
```
<x-simple-select
    name="permissions[]"
    id="permissions"
    multiple="multiple"
    :options="$permissions"
    value-field='id'
    text-field='name'
    :value="$selected"
    placeholder="Permissions"
    search-input-placeholder="Select Permissions..."
    :searchable="true"
/>
```

But the above code won't work in normal select, I try passing the id or the name directly but nothing would show.
```
<x-simple-select
    name="province"
    id="province"
    :options="$provinces"
    value-field='id'
    text-field='name'
    :value="$municipality->province_id"
    placeholder="Province"
    search-input-placeholder="Select Province..."
    :searchable="true"
/>
```

I'm still new to TALL so I don't understand anything when I read your code, wish you could help.

**Comment by victorybiz** [comment]
Can you please show an example of the data you passed to the `:options` prop.

From the sample you posted, your `$provinces` in `:options` should be array like;
```php
$provinces = [
    ['id' => '1', 'name' => 'Delta'],
    ['id' => '2', 'name' => 'Edo'],
    ['id' => '3', 'name' => 'Enugu'],
    ['id' => '4', 'name' => 'Lagos'],
];
```
and the selected value in `$municipality->province_id` should be a value of the province ID  e.g `2 for  for `Edo`

if you are using the `simple-select` in Livewire component, use `wire:model` instead of `:value`

**Comment by Alezter** [comment]
this is how I fill the options.
```
$provinces = Province::orderBy('name')->get()->toArray();
```

**Comment by Alezter** [comment]
I somehow found the problem, when I copy your example it work as you said but when I change the `id` from `string` to `int` it doesn't work anymore, since the `id` on my `$provinces` is `int`.

**Comment by victorybiz** [comment]
I just fixed this issue in [v1.2.5](https://github.com/victorybiz/laravel-simple-select/releases/tag/v1.2.5), you can now use `int` as `id`.

**Comment by Alezter** [comment]
Thank you! Working perfectly now.
